src/pages/Lending/Pool/components/Borrow/BorrowConfirm.tsx

```diff
@@ -546,7 +537,6 @@ export const BorrowConfirm = ({
                     loanTxStatus === 'pending' ||
                     isLoanPending ||
                     isEstimatedFeesDataLoading ||
-                    isEstimatedFeesDataError ||
```

Comment:
just making note this changes behavior slightly. not using this state elsewhere, so probably safe to remove though